spec/models/customer.spec.js

```diff
@@ -0,0 +1,72 @@
+var app = require('../../app')
+var db = app.get('db')
+var Customer = require('../../models/customer')
+
+describe('Customer', function () {
+  beforeEach(function(){
+
+  })
+
+  afterEach(function () {
+    db.end()
+  })
+
+
+  describe('#all', function () {
+    it('should return an array of objects', function(done) {
+      Customer.all(function(error,customers){
+        expect(customers).toEqual(jasmine.any(Array))
+        done()
+      })
```

Comment:
Also include a test that each element in the array is an object, as your test statement describes.